### DISCUSSION TITLE
ExplorerPatcher runed and tested on build 22489

### DISCUSSION THREAD

**Discussion by rxboy1983** [valinet/ExplorerPatcher#219] [General]
1. Icon of button "Start" is geted from Windows 10 style and not from Windows 11 style.
I need to edit .MSSTYLE file - TaskbarPearl>PEARLICON>IMAGEFILE1.
2. I set - taskbar icon size: Large (default). Icons are still to small for me.
3. TaskbandExtendedUI has not rounded corners. That same with button "Close". Checked it with custom theme and oryginal theme.
4. If I restart File Explorer then TaskbarX is automatically turned off.
5. Option "Show Windows build info on desktop" not work for me. It maybe because I use dev build.
6. Option "Center tray icon pop-up menus" is for what...? I do not see any changes...
7. Not work all changed settings for AccentPalette. It because ExplorePatcher has no option "Colorize everything with accent color" like in StartAllBack...?

**Comment by valinet** [comment]
Not one of the "issues" here represent a problem with the EP per se. And the tone of this post is infuriating to say the least.

Let's take number 1. First of all, this was never claimed to be a feature. Getting past that, I am actually interested in implementing such functionality. Instead of "I need" as if its a nuisance that the thing's not already there, you could have written, for example, some more info regarding your experience about it. I never knew the Start button icon is in the theme file - TaskbarPearl - PEARLICON... That's a good start to look for hints about it in `explorer.exe`. Of course, if you took a bit of time and investigated what the mechanism for loading that programmatically is, it would ease my job in eventually implementing the Start orb change functionality. I am not an expert in everything, I still learn A LOT of stuff every day and there's still A LOT of stuff I don't know about. Without any guidance, of course, I will at some point take the supplementary time to learn this proper and maybe that feature sees the light of day someday (at the moment, I am incredibly busy with other stuff, fortunately/unfortunately...).

Throwing a list of "non-working" stuff which more than half if not all of it does not even represent issues with this app is not very helpful, y'know... I appreciate the information given in #1 as well, thank you for that, but if you also took the time to look a bit deeper into it, that would have been even better.

Thanks
